packages/ignite-basic-structure/templates/Unholy/Containers/LoginScreen.js

```diff
@@ -112,7 +112,7 @@ class LoginScreen extends React.Component {
     const editable = !fetching
     const textInputStyle = editable ? Styles.textInput : Styles.textInputReadonly
     return (
-      <ScrollView contentContainerStyle={{justifyContent: 'center'}} style={[Styles.container, {height: this.state.visibleHeight}]} keyboardShouldPersistTaps>
+      <ScrollView contentContainerStyle={{justifyContent: 'center'}} style={[Styles.container, {height: this.state.visibleHeight}]} keyboardShouldPersistTaps='always'>
```

Comment:
Is this something new in 0.40?  BC, I've been teasing a short blog on the crazy prop for a while now, and if this is new, then blog++;